modules/nf-core/sentieon/tnhaplotyper2/tests/main.nf.test

```diff
@@ -0,0 +1,378 @@
+nextflow_process {
+
+    name "Test Process SENTIEON_TNHAPLOTYPER2"
+    config "./nextflow.config"
+    script "../main.nf"
+    process "SENTIEON_TNHAPLOTYPER2"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "picard"
+    tag "picard/addorreplacereadgroups"
+    tag "sentieon"
+    tag "sentieon/tnhaplotyper2"
+    config './nextflow.config'
+
+    setup {
+        run("PICARD_ADDORREPLACEREADGROUPS", alias: "NORMAL_ADDORREPLACEREADGROUPS") {
+            script "../../../picard/addorreplacereadgroups/main.nf"
+            config './nextflow.config'
+            process {
+                """
+                input[0] = [[id:'test'],// meta map
+                    [file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test.paired_end.recalibrated.sorted.bam',checkIfExists:true)]
+                    ]
+                input[1] = [[],[]]
+                input[2] = [[],[]]
+                """
+            }
+        }
+
+        run("PICARD_ADDORREPLACEREADGROUPS", alias: "TUMOUR_ADDORREPLACEREADGROUPS") {
+            script "../../../picard/addorreplacereadgroups/main.nf"
+            config './nextflow.config'
+
+            process {
+                """
+                input[0] = [[id:'test'],// meta map
+                    [file(params.modules_testdata_base_path + 'genomics/homo_sapiens/illumina/bam/test2.paired_end.recalibrated.sorted.bam',checkIfExists:true)]
+                    ]
+                input[1] = [[],[]]
+                input[2] = [[],[]]
+                """
+            }
+        }
+    }
+
+    test("no pon") {
```

Comment:
```suggestion
    test("homo_sapiens - bam - genome - germline - no pon") {
```

Reply:
I see no real benefit from changing the names of the tests like this, it is just to fit a completely arbitrary structure.